scripts/ci-e2e.sh

```diff
@@ -155,35 +170,42 @@ BMO_OVERLAYS=(
   "${REPO_ROOT}/test/e2e/data/bmo-deployment/overlays/pr-test"
   "${REPO_ROOT}/test/e2e/data/bmo-deployment/overlays/release-latest"
 )
-IRONIC_OVERLAYS=(
-  "${REPO_ROOT}/test/e2e/data/ironic-deployment/overlays/release-26.0"
-  "${REPO_ROOT}/test/e2e/data/ironic-deployment/overlays/release-27.0"
-  "${REPO_ROOT}/test/e2e/data/ironic-deployment/overlays/release-29.0"
-  "${REPO_ROOT}/test/e2e/data/ironic-deployment/overlays/pr-test"
-  "${REPO_ROOT}/test/e2e/data/ironic-deployment/overlays/release-latest"
+IRSO_IRONIC_OVERLAYS=(
+  "${REPO_ROOT}/test/e2e/data/ironic-standalone-operator/ironic/overlays/release-26.0"
+  "${REPO_ROOT}/test/e2e/data/ironic-standalone-operator/ironic/overlays/release-27.0"
+  "${REPO_ROOT}/test/e2e/data/ironic-standalone-operator/ironic/overlays/release-29.0"
+  "${REPO_ROOT}/test/e2e/data/ironic-standalone-operator/ironic/overlays/release-main"
+  "${REPO_ROOT}/test/e2e/data/ironic-standalone-operator/ironic/overlays/pr-test"
 )
 
 # Update BMO and Ironic images in kustomization.yaml files to use the same image that was used before pivot in the metal3-dev-env
 case "${REPO_NAME:-}" in
   baremetal-operator)
     # shellcheck disable=SC2034
-    BARE_METAL_OPERATOR_IMAGE="${REGISTRY}/localimages/tested_repo:latest"
+    export BARE_METAL_OPERATOR_IMAGE="${REGISTRY}/localimages/tested_repo:latest"
     ;;
 
   ironic-image)
     # shellcheck disable=SC2034
-    IRONIC_IMAGE="${REGISTRY}/localimages/tested_repo:latest"
+    export IRONIC_IMAGE="${REGISTRY}/localimages/tested_repo:latest"
     ;;
 esac
 
+IRONIC_IMAGE="${IRONIC_IMAGE:-quay.io/metal3-io/ironic:main}"
 update_kustomize_image quay.io/metal3-io/baremetal-operator BARE_METAL_OPERATOR_IMAGE "${REPO_ROOT}"/test/e2e/data/bmo-deployment/overlays/pr-test
-update_kustomize_image quay.io/metal3-io/ironic IRONIC_IMAGE "${REPO_ROOT}"/test/e2e/data/ironic-deployment/overlays/pr-test
 
 # Apply envsubst to kustomization.yaml files in BMO and Ironic overlays
 kustomize_envsubst "${REPO_ROOT}"/test/e2e/data/bmo-deployment/overlays/pr-test
-kustomize_envsubst "${REPO_ROOT}"/test/e2e/data/ironic-deployment/overlays/pr-test
-
-# Create usernames and passwords and other files related to ironi basic auth if they
+kustomize_envsubst "${REPO_ROOT}"/test/e2e/data/ironic-standalone-operator/operator
+kustomize_envsubst "${REPO_ROOT}"/test/e2e/data/ironic-standalone-operator/ironic/overlays/release-26.0
```

Comment:
26.0 to 29.0 are out of support btw, and IRSO will drop them:
https://github.com/metal3-io/ironic-standalone-operator/pull/380

if we need to use those, then we need to use IRSO release branches.